ci/infra/azure/bastion-instance.tf

```diff
@@ -0,0 +1,12 @@
+resource "azurerm_bastion_host" "bastionhost" {
+  count               = var.create_bastionhost ? 1 : 0
+  name                = "${var.stack_name}-bastion"
+  location            = azurerm_resource_group.resource_group.location
+  resource_group_name = azurerm_resource_group.resource_group.name
+
+  ip_configuration {
+    name                 = "configuration"
```

Comment:
Not sure if we need some meaningful name here not just the same example from the terraform doc. Should we?

Reply:
changed